applications/callflow/src/module/cf_park.erl

```diff
@@ -732,23 +722,22 @@ get_endpoint_id(Username, Call) ->
 %% Ringback the device that parked the call
 %% @end
 %%--------------------------------------------------------------------
--spec ringback_parker(api_binary(), ne_binary(), ne_binary(), kz_json:object(), kapps_call:call()) ->
+-spec ringback_parker(api_binary(), ne_binary(), kz_json:object(), kapps_call:call()) ->
                              'answered' | 'failed' | 'channel_hungup'.
-ringback_parker('undefined', _, _, _, _) -> 'failed';
-ringback_parker(EndpointId, SlotNumber, TmpCID, Data, Call) ->
+ringback_parker('undefined', _, _, _) -> 'failed';
+ringback_parker(EndpointId, SlotNumber, Data, Call0) ->
+    TmpCID = <<"Parking slot ", SlotNumber/binary>>,
+    Call = kapps_call:kvs_store('dynamic_cid', {'undefined', TmpCID}, Call0),
```

Comment:
when ringing back the parker, we want to keep the initial number but change the `display`